tiled/src/main/kotlin/ktx/tiled/mapObjects.kt

```diff
@@ -120,3 +121,13 @@ val MapObject.shape: Shape2D
     is RectangleMapObject -> rectangle
     else -> throw MissingShapeException("MapObject of type ${this::class.java} does not have a shape.")
   }
+
+/**
+ * Returns **true** if and only if the [MapObjects] collection is empty.
+ */
+fun MapObjects.isEmpty() = this.count <= 0
```

Comment:
If it's possible for the tiled map to return a null instead of `MapObjects`, this will be a bit more useful:
```kotlin
@OptIn(ExperimentalContracts::class)
fun MapObjects?.isEmpty(): Boolean {
  contract {
    returns(false) implies (this@isEmpty != null)
  }
  return this == null || this.count <= 0
}
```
This function, along with `isNotEmpty`, would work both as a null check and a count check. If it's never null, feel free to resolve this comment. Otherwise, the same applies to `isNotEmpty` and map layers.

Reply:
A `Map` in LibGDX always comes with an initialized `MapLayers` and `MapProperties` object. There is no setter from what I can tell. Therefore, null should not be possible for `MapLayers` unless you hack yourself in with reflection.

The same is true for `MapLayer` and its `MapObjects` property.

Should we consider the reflection use case? I personally wouldn't do it because when you hack around like that you are on your own ;)

```Java
public class Map implements Disposable {
	private MapLayers layers = new MapLayers();
	private MapProperties properties = new MapProperties();
```

```Java
public class MapLayer {
	private String name = "";
	// ...
	private MapObjects objects = new MapObjects();
	private MapProperties properties = new MapProperties();
```

Reply:
> Should we consider the reflection use case? I personally wouldn't do it because when you hack around like that you are on your own ;)

No, it's OK. No need to be paranoid. :)